x/blob/test/decode_blob_tx_test.go

```diff
@@ -0,0 +1,111 @@
+package test
+
+import (
+	"crypto/sha256"
+	"encoding/base64"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/celestiaorg/celestia-app/app"
+	"github.com/celestiaorg/celestia-app/app/encoding"
+	"github.com/celestiaorg/celestia-app/pkg/blob"
+	"github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/assert"
+)
+
+type BlockResponse struct {
+	Block Block `json:"block"`
+}
+
+type Block struct {
+	Data BlockData `json:"data"`
+}
+
+type BlockData struct {
+	Txs []string `json:"txs"`
+}
+
+// TestDecodeBlobTx demonstrates how one can take the response from the Comet
+// BFT API endpoint /cosmos/base/tendermint/v1beta1/blocks/{block_number} and
```

Comment:
[Question] Why isn't it named "Celestia-core API" instead of "CometBFT API"? This choice might be leading to confusion, as readers could associate it with Comet-BFT APIs (while they are very similar, yet they have their own differences like this one) .

Reply:
dd6e0d829e8b72b1c4776ef52a4d0d69dd023516